src/content/en/tools/chrome-devtools/beginners/devtools-for-beginners1.md

```diff
@@ -0,0 +1,239 @@
+
+This is the first of three DevTools For Beginners tutorials. 
+These tutorials walk you through basic web fundamentals 
+as well as how to improve your design productivity with DevTools. 
+You will be constructing a personal website from scratch,
+and the end product will look something like 
+[this](https://dfb5.glitch.me/). 
+
+## Setup {: #setup}
+
+In order to start creating your site, you need to set up your code:
+
+1. Open the source code. A code editor called Glitch shows a page called 
+index.html. The HTML is mostly empty. You'll be adding your own code to 
+it.
+<figure> <img src="init.png" alt="Initial Source Code Window" width="auto" height="auto">
+<figcaption>
+    <b>Figure 1</b>. The initial window you will see.
+  </figcaption>
+</figure>
+2. Click **dfb-1**. A menu pops up.
+<figure> <img src="menu.png" alt="Glitch Remix Menu" width="auto" height="auto">
+<figcaption>
+    <b>Figure 2</b>. The Glitch menu that will appear.
+  </figcaption>
+</figure>
+
+3. Click **Remix**. Glitch creates a copy of the project that you can 
+edit. Note that the name of the new project will be randomly generated
+and not dfb-1.
+The content is the same, but the name on the top-left has changed.
+4. Click Show Live. Another tab opens with view of what your site 
+currently looks like.
+
+<figure> <img src="2tabs.png" alt="An image of the two open tabs" width="auto" height="auto">
+<figcaption>
+    <b>Figure 3</b>. The viewing and editing tabs.
+  </figcaption>
+</figure>
+
+Now, you have two tabs open: the code
+(which will be called the editing tab) 
+and the preview of your website (which will be called the viewing tab). 
+
+## Adding Content {: #adding-content}
+
+Since your website is pretty empty, let’s add some content to it! 
+In the editing tab, you’ll see two sections in your HTML document: 
+the head (the text between `<head>` and `</head>`) and the body 
+(the text between `<body>` and `</body>`). 
+The head contains the metadata of your site, which is invisible 
+data your web browser parses to understand the content of your site. 
+This helps search engines find your pages, among other things. 
+The body contains the actual content of your page, like text or images.
+You won’t be adding much in the `<head>` section in this part of the 
+tutorial, so for now, let’s move on to the fun part, the `<body>` 
+section. Since this is a personal website, you’ll definitely want to 
+have something about yourself in there. Many personal websites have an
+‘About Me’ section, and yours will as well. Here’s how you’ll do it:
+
+1. Go to line 19 and press <kbd>Enter</kbd>
+2. You want a heading for this section, 
+so type `<h1>About Me</h1>`. 
+This formats the text as a heading. 
+3. To see the changes, go to the viewing tab.
+4. Go back to the editing tab.
+5. You’ll want to put in text, too, so insert a `<p>` element, 
+like `<p> I am learning HTML.</p>` under the line where you put 
+the heading. 
+6. Go to the viewing tab.
```

Comment:
Is it necessary to go back to the viewing tab and then the editing tab or can you just go to editing tab?